src/util/misc/planeChange.ts

```diff
@@ -104,10 +113,8 @@ export const transformPointRelativeToCanvas = (
  * @param {Matrix} [to] destination plane matrix to contain object. Passing `null` means `object` should be sent to the canvas coordinate plane.
  * @returns {Matrix} the transform matrix that was applied to `object`
  */
-export const sendObjectToPlane = (object: any, from: TMat2D = iMatrix, to: TMat2D = iMatrix): TMat2D => {
-  //  we are actually looking for the transformation from the destination plane to the source plane (which is a linear mapping)
-  //  the object will exist on the destination plane and we want it to seem unchanged by it so we reverse the destination matrix (to) and then apply the source matrix (from)
-  const t = multiplyTransformMatrices(invertTransform(to), from);
+export const sendObjectToPlane = (object: TObject, from: TMat2D = iMatrix, to: TMat2D = iMatrix): TMat2D => {
```

Comment:
I want to add a signature that accepts objects once we migrate `Object` so I can do `instanceof`